src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -57,13 +57,13 @@ public PravegaStorageDriver(
 			throws OmgShootMyFootException {
 		super(testStepId, dataInput, storageConfig, verifyFlag, batchSize);
 		this.uriSchema = uriSchema;
-		streamManager = StreamManager.create(URI.create(uriSchema));
+
 
 		final String uid = credential == null ? null : credential.getUid();
 		final Config nodeConfig = storageConfig.configVal("net-node");
 		nodePort = storageConfig.intVal("net-node-port");
 		final List<String> endpointAddrList = nodeConfig.listVal("addrs");
-		readerTimeoutMs = storageConfig.intVal("storage-item-input-readerTimeout");
+		readerTimeoutMs = storageConfig.intVal("item-input-readerTimeout");
```

Comment:
What do you think about to use `storage-read-timeoutMillis` instead `storage-item-input-readerTimeout`?

Reply:
Do you suggest changing the path in JSON file or do you want to say I should use the "storage-net-timeoutMillis" JSON parameter? If it's the second one, then sure, I'll change it, I just wasn't sure if the "net-timeoutMillis" had something to do with the reader. If it's the first one, I'll correct it as well, sounds good to me.

> What do you think about to use `storage-read-timeoutMillis` instead `storage-item-input-readerTimeout`?

Do you suggest changing the path in JSON file or do you want to say I should use the "storage-net-timeoutMillis" JSON parameter? If it's the second one, then sure, I'll change it, I just wasn't sure if the "net-timeoutMillis" had something to do with the reader. If it's the first one, I'll correct it as well, sounds good to me.

Reply:
I mean a new configuration option `storage-read-timeoutMillis` describing the Pravega-specific & read-specific timeout